mantle/cmd/kola/qemuexec.go

```diff
@@ -192,6 +193,7 @@ func runQemuExec(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return err
 	}
+	defer inst.Destroy()
```

Comment:
It turns out we were leaving a bunch of swtpm defunct processes laying around. I had ~200 on my system.

Reply:
This is probably right, but we use `prctl(PR_SET_PDEATHSIG)` though which should have prevented that anyways.